api/src/opentrons/protocol_engine/commands/absorbance_reader/read.py

```diff
@@ -18,16 +18,17 @@
 
 
 class ReadAbsorbanceParams(BaseModel):
-    """Input parameters for a single absorbance reading."""
+    """Input parameters for an absorbance reading."""
 
     moduleId: str = Field(..., description="Unique ID of the Absorbance Reader.")
-    sampleWavelength: int = Field(..., description="Sample wavelength in nm.")
 
 
 class ReadAbsorbanceResult(BaseModel):
-    """Result data from running an aborbance reading, returned as a dictionary map of values by well name (eg. ("A1": 0.0, ...))."""
+    """Result data from running an aborbance reading, returned as a dictionary map of wavelenths containing a map of values by well name (eg. {450: {"A1": 0.0, ...}})."""
```

Comment:
```suggestion
    """Result data from running an aborbance reading, returned as a dictionary map of wavelengths containing a map of values by well name (eg. {450: {"A1": 0.0, ...}})."""
```
Set up a text replacement maybe 😆